docs/basics/101-5-install.rst

```diff
@@ -0,0 +1,204 @@
+
+Note: if we had not specified the path ``books/ml-books``, the command would have installed the
+dataset in the root of the directory and used the name of the remote repository
+"``machinelearning-books``". Alternatively, you could have also installed the dataset from within
+the books directory. Important: If you are installing *within* a dataset,
+but execute the command not from the root of this dataset, your ``-d`` option needs to specify
+the path to the root of the dataset, and the path needs to start from the directory root. For example:
+``datalad install -d ../ -s https://github.com/datalad-datasets/machinelearning-books.git books/ml-books``
+(with ``../`` being the Unix expression for ``parent directory``, i.e. "one-directory-up").
+
+.. container:: toggle
+
+   .. container:: header
+
+       **Addition: What if I don't install into an existing dataset?**
+
+   If you don't install inside an existing dataset, you only need to omit the ``dataset``
+   option. You can try ``datalad install -s https://github.com/datalad-datasets/machinelearning-books.git``
+   anywhere outside of your ``Datalad-101`` dataset to install the dataset into a new directory
+   called ``machinelearning-books``.
+
+.. gitusernote::
+
+   The ``datalad install`` command uses ``git clone``.
+
+Here is the repository structure:
+
+.. runrecord:: _examples/DL-101-5-2
+   :language: console
+   :workdir: dl-101/DataLad-101
+
+   $ tree
+
+We can see that books contains the three PDFs we downloaded and saved previously.
+Additionally, there is another directory ``ml-books`` which contains 10 more books,
+and a ``README.md`` file.
+You might be confused by the small arrows and cryptic paths that follow every PDF -
+we'll get to what that is in the next section.
+
+
+Dataset content identity and availability information
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+You might have been surprised by how fast the datalad was installed. Shouldn't
+a download of many books should take much more time? Here you can see another
+import feature of DataLad datasets and the ``datalad install`` command:
+
+Upon installation of a DataLad dataset, DataLad retrieves only small files
+(for example text files or markdown files) and (small) metadata
+information about the dataset. It does not, however, download any large files
+(yet). The metadata exposes the datasets file hierarchy
+for exploration, and downloading only this speeds up the installation of a DataLad dataset
+of many TB in size to a few seconds. Just now, after installation, the dataset is
+small in size:
+
+.. runrecord:: _examples/DL-101-5-3
+   :language: console
+   :workdir: dl-101/DataLad-101/
+
+   $ cd books/ml-books
+   $ du -sh      # Unix command to show size of contents
+
+This is tiny. So lets see whats in this dataset.
+
+.. runrecord:: _examples/DL-101-5-4
+   :language: console
+   :workdir: dl-101/DataLad-101/books/ml-books
+
+   $ ls
+
```

Comment:
This might be the place to introduce `datalad status --annex basic`, but https://github.com/datalad/datalad/issues/3519 :(

Reply:
oh this is such a cool command! :(
We could however switch datasets to the longnow podcast recordings? As far as I can see, this dataset does not have URL-type annex keys. The ML books weren't 100% fitting into the narrative, so we can equally well just pretend that the podcasts are audio recordings of a past years course (or something similar). What do you think?

Reply:
That would be great! I think that dataset is a good fit for an educational narrative. We could later on even reveal that it is auto-generated by monitoring a podcast feed:

```
git annex importfeed --template '${feedtitle}/${itempubdate}__${itemtitle}${extension}' --force http://longnow.org/projects/seminars/SALT.xml
```

Which would be a straightforward connection to a potential future chapter on getting non-DataLad data sources into the DataLad world (scraping, crawling, ...)

Reply:
Very good idea, I like this! I'll rewrite this then.

Reply:
Cool. I have just pushed an update of this dataset. I have used `datalad run` this time. So there is now the possibility to have readers `rerun` the update and obtain any new episodes....

Reply:
awesome!!!